protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/ToConnectTestDataGenerator.java

```diff
@@ -253,6 +256,87 @@ public static Struct getPrimitiveTypesData(String packageName) {
             .build();
     }
 
+    public static List<Message> getEnumProtobufMessages() {
+        return Arrays.asList(
+                EnumTypeSyntax3.EnumTest.newBuilder()
```

Comment:
Can you also test if we do not set every enum value in the conversion? Is it working?

Reply:
Tested unsetting optional enum with default value, works fine. I will leave the defaultValue test unset.